ui/src/app/components/update-spanner-config-form/update-spanner-config-form.component.html

```diff
@@ -7,9 +7,17 @@ <h2>Connect to Spanner</h2>
       <input matInput placeholder="project id" type="text" formControlName="GCPProjectID" />
     </mat-form-field>
     <br />
-    <mat-form-field class="full-width" appearance="outline">
+    <mat-form-field class="full-width"  
+    hintLabel="Min. 2 characters and Max. 64 characters" 
+    appearance="outline">
       <mat-label>Instance ID</mat-label>
-      <input matInput placeholder="instance id" type="text" formControlName="SpannerInstanceID" />
+      <input matInput placeholder="instance id" type="text" 
+      required minlength="2"
```

Comment:
Since the instance ID should be 4-30 character long and have only Lowercase letters, numbers, hyphens allowed. Lets keep the check as that

Reply:
Resolved

Reply:
modify the minimum and maximum length as well

Reply:
Updated

Reply:
i dont see the length modified ?